gui/src/components/Taipy/Chart.tsx

```diff
@@ -212,12 +209,16 @@ const Chart = (props: ChartProp) => {
     useEffect(() => {
         if (refresh || !data[dataKey.current]) {
             const backCols = Object.keys(config.columns).map((col) => config.columns[col].dfid);
+            const xAxis = layout.xaxis.title instanceof Object && "text" in layout.xaxis.title ? layout.xaxis.title.text : layout.xaxis.title;
+            const yAxis = layout.yaxis.title instanceof Object && "text" in layout.yaxis.title ? layout.yaxis.title.text : layout.yaxis.title;
```

Comment:
I guess we could also define a "sub-payload" for decimator so that we don't calculate things if there's no decimator
ie 
```
const decimatorPayload = decimator ? {
  width: plotRef.current?.clientWidth,
  xAxis: config.traces.length && config.traces[0].length && config.traces[0][0] && config.columns[config.traces[0][0]].dfid,
   yAxis: config.traces.length == 1 && config.traces[0].length > 1 && config.columns[config.traces[0][1]] && config.columns[config.traces[0][1]].dfid,
   decimator: decimator
} : undefined;
```
Then pass this as a decimator parameter to createRequestChartUpdateAction ?